corehq/form_processor/models.py

```diff
@@ -1131,7 +1032,6 @@ class CaseTransaction(PartitionedModel, SaveStateMixin, models.Model):
     TYPE_CASE_CREATE = 128
     TYPE_CASE_CLOSE = 256
     TYPE_CASE_INDEX = 512
-    TYPE_CASE_ATTACHMENT = 1024
```

Comment:
Probably worth keeping this and making a comment that the supporting functionality has been removed since we likely have some test data that uses this value.